iidm/iidm-util/src/main/java/com/powsybl/iidm/network/util/TerminalChooser.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.util;
+
+import com.powsybl.iidm.network.ConnectableType;
+import com.powsybl.iidm.network.Injection;
+import com.powsybl.iidm.network.Terminal;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+/**
+ * A class which allows to choose a terminal, according to given filters, among several terminals
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public final class TerminalChooser {
+
+    private final List<Predicate<Terminal>> rules;
+
+    public static TerminalChooser getDefaultSlackTerminalChooser() {
```

Comment:
I would rename this method `getDefault` to avoid redundancy between the name of the class, and the name of this method